trestle/core/catalog_interface.py

```diff
@@ -525,6 +525,20 @@ def _get_full_profile_param_dict(profile: prof.Profile) -> Dict[str, common.Para
             set_param_dict[set_param.param_id] = param
         return set_param_dict
 
+    @staticmethod
+    def get_profile_displayname_param_dict(profile: prof.Profile) -> Dict[str, str]:
+        """Get the mapping of param_id to display-name set in the properties of set-params."""
+        displayname_dict: Dict[str, str] = {}
+        if not profile.modify:
+            return displayname_dict
+
```

Comment:
could do: if profile.modify so there is only one return

Reply:
done